Library/Homebrew/livecheck/strategy/electron_builder.rb

```diff
@@ -19,7 +19,7 @@ class ElectronBuilder
         PRIORITY = 0
 
         # The `Regexp` used to determine if the strategy applies to the URL.
-        URL_MATCH_REGEX = %r{^https?://.+/.+\.ya?ml$}i.freeze
+        URL_MATCH_REGEX = %r{^https?://.+/.+\.ya?ml(\?.+=.+)?$}i.freeze
```

Comment:
```suggestion
        URL_MATCH_REGEX = %r{^https?://.+/[^/]+\.ya?ml(?:\?[^/?]+)?$}i.freeze
```

Though less common, it's technically possible to have query string parameter fields without values (e.g., `?a&b` instead of `?a=1&b=2`), so It may be better for us to be a little looser about what comes after the `?`. Matching anything except a forward slash or question mark is probably sufficient.

Past that, I've made the group non-capturing (i.e., `(?:)`), as that tends to be standard within livecheck unless we specifically need to capture something in a regex. Additionally, this replaces `/.+` with `/[^/]+`, to better align with `URL_MATCH_REGEX` in other strategies.

Reply:
Thanks @samford, it was meant to be non-capturing, I just didn't type what I was thinking in my head 😄